remove dependencies

hugoduraes opened the pull request:
Fixes #1

hugoduraes committed:
remove dependencies

AlphaT3ch commented:
@hugoduraes Sorry to bother you again (great PR by the way), but what I meant is you need on refork the repo so the Travis CI files are the same on both branches

hugoduraes commented:
I've fetched it from your repo and rebased my changes. So the files should've been the same.
Nonetheless, I'm glad to know you merged it and fixed the issue.
